lib/cookpad_mysql_defaults.rb

```diff
@@ -1,6 +1,4 @@
 require "cookpad_mysql_defaults/mysql_table_default_options"
-require "cookpad_mysql_defaults/table_definition"
-require "cookpad_mysql_defaults/schema_statements"
```

Comment:
I don't think reverting this is a good idea, since this may break other Rails 6 app that relies on this flag too.

What about requiring the code if AR version is less than 7?

```ruby
if ActiveRecord::VERSION::MAJOR < 7
  require "cookpad_mysql_defaults/table_definition"
  require "cookpad_mysql_defaults/schema_statements"
end
```